ldap/servers/slapd/back-ldbm/db-mdb/mdb_import.c

```diff
@@ -736,7 +736,11 @@ dbmdb_import_all_done(ImportJob *job, int ret)
             ldbm_set_last_usn(inst->inst_be);
 
             /* bring backend online again */
-            slapi_mtn_be_enable(inst->inst_be);
+            if (job->flags & FLAG_REINDEXING) {
+                instance_set_not_busy(inst);
```

Comment:
Fix looks good. A bit painful to retrieve where it was set, I found bdb_db2index and dbmdb_db2index. Do you mind to put a comment from where the BUSY flag was set.

Reply:
Ok, I will add a comment

Reply:
Done.